app/src/main/java/com/miraclehwan/detektintegrationexample/FailTest.kt

```diff
@@ -0,0 +1,9 @@
+package com.miraclehwan.detektintegrationexample
+
+data class FailTest (val a:Int,val  b:Boolean,
+
+                     val c:Int){
+    fun testtest( a:Int) {}
+
+    fun maxLineWarning_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890( a:Int) = Unit
```

Comment:
⚠️ **[detekt]** <detekt.FunctionNaming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function names should match the pattern: [a-z][a-zA-Z0-9]*